bin/build_utils_gitlab

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+[[ "$TRACE" ]] && set -x
```

Comment:
`[[ ]] && <something>`
should probably be
```
if [[ ]]; then
  <something>
fi
```
as the latter is much more readable

Reply:
agreed